src/MultiOwnable.sol

```diff
@@ -120,8 +120,7 @@ contract MultiOwnable {
     /// @param index The index of the owner to be removed.
     /// @param owner The ABI encoded bytes of the owner to be removed.
     function removeOwnerAtIndex(uint256 index, bytes calldata owner) external virtual onlyOwner {
```

Comment:
These methods seem like overkill as they require the caller to pass in both the owner and index, when either should suffice.